### PR TITLE
paperclip installed

### PR DESCRIPTION
**Pull request by erikaquinn**
https://trello.com/c/AYEQJhpT/112-paperclip-install

able to upload company profile photo

### CODE REVIEW

File: app/models/company.rb [hunk 1]
```diff
@@ -99,6 +103,9 @@ class Company < ApplicationRecord
   has_many :reviews
   has_many :customers, through: :reviews
   has_many :quotes
+  
+  has_attached_file :avatar, styles: { medium: "300x300>", thumb: "100x100>" }, default_url: "/images/missing.png"
```
Comment (gatorjuice) [inline app/models/company.rb]: wondering if we should change this back to `default_url: "/images/:style/missing.png"`

Comment (erikaquinn) [inline app/models/company.rb]: Sure! We can do that! I'll be sure to make the changes this evening and add the pull request to Slack when it's ready.

Comment (gatorjuice) [inline app/models/company.rb]: @erikaquinn can you still push that change?

Comment (erikaquinn) [inline app/models/company.rb]: @gatorjuice Done!

**Pull request merged**